src/Lykke.Service.Assets.Repositories/Erc20Repository.cs

```diff
@@ -87,7 +75,7 @@ public async Task<IEnumerable<IErc20Token>> GetAllWithAssetsAsync()
 
             return (await GetByAssetIndexesAsync(indexes))
                 // Ensure, that our indexes are not corrupted. It's important, we have already faced problems with them.
-                .Where(x => x.AssetId != null);
+                .Where(x => x?.AssetId != null);
```

Comment:
`.Where(x => !string.InNullOrWhiteSpace(x?.AssetId);`
?

Reply:
I don't think such validation is appropriate here. Whitespace in AssetId is out-of-contracts problem.